internal/pkg/term/progress/cloudformation.go

```diff
@@ -96,6 +96,11 @@ func updateComponentTimer(mu *sync.Mutex, statuses []stackStatus, sw *stopWatch)
 	curStatus, nextStatus := statuses[len(statuses)-2], statuses[len(statuses)-1]
 	switch {
 	case nextStatus.value.InProgress():
+		// Reset and start only if the status moved to in progress from a static state.
```

Comment:
Does the status ever change in between IN_PROGRESS statuses? Or they're always back-to-back?

Reply:
yea it can,

for example, this scenario can happen: "not started" -> "create in progress" -> "create complete" -> "delete in progress" -> "delete complete"

So between the "create in progress" and "delete in progress", the creation was successful. However, another resource failed so a deletion triggered.

This scenario is fine tho for us, because the timer will be stopped and then called reset in this case-statement